docs/source/markdown/podman-untag.1.md

```diff
@@ -0,0 +1,37 @@
+% podman-untag(1)
+
+## NAME
+podman\-untag - Removes one or more names from a locally-stored image
+
+## SYNOPSIS
+**podman untag** *image*[:*tag*] *target-names*[:*tag*] [*options*]
+
+**podman image untag** *image*[:*tag*] *target-names*[:*tag*] [*options*]
+
+## DESCRIPTION
+Removes one or all aliases of an image. An alias refers to the entire image
```

Comment:
"aliases" versus "names" - I prefer sticking with names to remain consistent

Reply:
Yes, changed it to `name`